lib/dj/engine/stateMachine.ts

```
@@ -0,0 +1,463 @@
+      return handlePlayingEvent(state, event);
+    case 'cueing':
+      return handleCueingEvent(state, event);
+    case 'planned':
+      return handlePlannedEvent(state, event);
+    case 'crossfading':
+      return handleCrossfadingEvent(state, event);
+    case 'paused':
+      return handlePausedEvent(state, event);
+    default:
+      // Exhaustive check
+      const _exhaustive: never = state;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)